test/python/rest/string_tests.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
What is the purpose of `string_tests` in the `rest` directory? It doesn't test any rest functionality.

Reply:
This comes from the example in the README file. It didn't work, so I copied the example into this source file, fixed it, tested it, and copied it back into the README file. I could delete the source file, but think working examples are more useful than commentary examples, so I'd rather replace the example in the README file with a link to this source file.